src/tools/pmempool/transform.c

```diff
@@ -171,8 +176,12 @@ pmempool_transform_func(char *appname, int argc, char *argv[])
 	if ((ret = pmempool_transform_parse_args(&ctx, appname, argc, argv)))
 		return ret;
 
-	ret = pmempool_transform(ctx.poolset_file_src, ctx.poolset_file_dst,
-			ctx.flags);
+	/* check whether progress should be reported */
+	if (util_is_pmempool_progress_set())
```

Comment:
So the env variable is just an alternative mechanism for enabling progress reporting? I don't quite understand why.
I agree /w Krzysiek that this should be on by default and disabled a flag.